data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,672 @@
+			choice
+				`	"So close to what?"`
+			`	"To changing everything, to remaking the Syndicate, to creating a whole new political order."`
+			`	She wipes away a tear and visibly steels herself to speak, "I was Alastair's chosen successor, I had already dealt with the competition, a few more years and I'd have been a hero!"`
+			`	She turns slightly and gestures at the sea. "Instead now I'm here, and you get to be the hero, a dumb war hero," her voice rises in pitch and volume, "Never mind a brighter future for humanity!"`
+			`	Her expression seems to be wrestling between fury and despair at this point, "So now what? You here to kill me then?!"`
+			choice
+				`	"You kidnapped thousands of people, how is that a brighter future?"`
+			`	"Of course you have no idea!" she throws up her hands and expresses a noise of frustration.`
+			`	Her voice drops to a more reasonable level. "I never wanted to kidnap people," and she gets quieter again, "I inherited that legacy..."`
+			`	"You still profited off it, did nothing to stop it," you say.`
+			`	Your tone had not been accusatory, but merely matter of fact, and yet they seem to have struck her like a sack of bricks. Her composure shatters and she begins to sob, sinking to her knees and shielding her face from the wind.`
+			`	"It was never meant to be like this," she forces out, "but the Philosophers were our only resource, and we weren't in control anymore."`
+			`	"We wanted to reform the government, make it better, make it work for the people instead of against them. We dealt with the devil to keep that dream alive I suppose. I guess I should have expected it to get us burnt eventually."`
+			choice
+				`	"Who is 'us'?"`
+			`	She sighs, "My father and I, and a couple of the old survivors of the purge. It's only me left now though, just one Base Principle."`
+			`	You shake your head and ask, "Do you realize you nearly started a war that would have killed millions? Possibly ended human government entirely!"`
+			`	"I know," she says, calm now, "obviously not my intent, but maybe that would have been better in the long-run than the Republic and its corruption. There's no apology appropriate for it, and I can't pretend it would be totally sincere on that front. War changes things, sometimes some pain in the short term is needed for a better future."`
+			`	"The Free Worlds were almost the perfect catalyst, but none of us could have predicted the Pug. Still, that nearly forced Alastair into retirement, it was almost enough. Were it not for the Pug I would already be running the Syndicate, and there would have been no further risk with the Hai. None of this would have ever happened."`
+			`	Her expression is defiant, "No one would have cared about a few missing people in the world I would have built! A necessary sacrifice in the name of progress, and I would've shut down the pirates myself."`
+			choice
+				`	"It's always fine to say as such about plans that didn't happen."`
```

Comment:
Yes, basically. It's easy for people to assume that everyone would see things their way if the improvements they sought actually happened. People, of course, aren't typically that uniform in response to things, even if it benefits them.

Reply:
```suggestion
				`	"It's easy to say that without seeing the end result. You can't know whether it would have worked out that way."`
```
This could maybe still use some improving, but I think it's more clear than it was previously. Thoughts?